src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly

```diff
@@ -58,8 +58,8 @@
             <st:nbsp/>${%No}
         </f:entry>
 
-        <f:entry title="${%Decline offer duration}" field="declineOfferDuration" description="${%Decline offers for specified amount of time in case no slave is queued. The plugin asks for new offers in case a new slave needs to be started. In milliseconds.}">
-          <f:number clazz="required positive-number" min="1000" steps="1000" default="600000"/>
+        <f:entry title="${%Decline offer duration}" field="declineOfferDuration" description="${%Decline offers for specified amount of time in case no slave is queued. The plugin asks for new offers in case a new slave needs to be started. In seconds.}">
```

Comment:
Could changing the unit here without a data migration cause issues for some users?

Reply:
Data migration is not required here because the change removed some restrictions instead of adding.